addons/account/static/src/xml/account_payment.xml

```diff
@@ -6,18 +6,22 @@
         <div>
             <t t-if="outstanding">
                 <div>
-                    <strong class="float-left" id="outstanding"><t t-esc="title"></t></strong>
+                    <strong class="float-left" id="outstanding_title"><t t-esc="title"></t></strong>
```

Comment:
`<strong class="float-left" id="outstanding_title" t-esc="title"/>`
Also, is the renaming to `outstanding_title` really necessary? I don't see any reference to it in your diff